frontend/src/components/dataset/FacetFilter.vue

```diff
@@ -151,19 +148,36 @@ export default{
             count: {},
             numFilters: 0,
             totalCount: 0,
-            maxFilters: 0,
             updates: 0,
         }
     },
 
     computed: {
         ...mapState({
             filtered: state => state.search.facets,
-            facet: state => state.dataset.facets,
+            facets: state => state.dataset.facets,
             open(state){
                 return state.dataset.facetOpen[this.field.name];
             }
         }),
+        maxFilters: function(){
+            var keys = Object.keys(this.filters);
+            let rv = 2;
+            for (var j=0; j<keys.length; j++){
+                rv += this.filters[keys[j]].length;
+            }
+            return rv;
+        },
+        filters: function(){
+            let filters = {};
+            for (let i=0; i<this.field.facets.length; i++){
+                let fk = Object.keys(this.field.facets[i])[0];
+                if ( (typeof(this.facets) === "object") && (typeof(this.facets[fk]) !== "undefined") ){
+                    filters[fk] = this.facets[fk][fk];
```

Comment:
Is this line correct?  Using the filter key twice looks odd (`this.facets[fk][fk]`).

Reply:
@BrandonSharratt - Can you confirm that this line is correct?